modules/management-vpc/README.md

```diff
@@ -6,6 +6,17 @@ You can use this submodule when you need more modularity to create your topology
 
 See the [Landing Zone example](../../examples/landing_zone/) for runnable code.
 
+### Usage
+```terraform
+module management_vpc {
+  source              = "terraform-ibm-modules/terraform-ibm-landing-zone-vpc/ibm//modules/management-vpc"
```

Comment:
sorry, source is still wrong here. should be `terraform-ibm-modules/landing-zone-vpc/ibm//modules/management-vpc`